website/blog/2023-03-03-databend-weekly-83.md

```diff
@@ -0,0 +1,182 @@
+- name: Carlosfengv
+  url: https://github.com/Carlosfengv
+  image_url: https://github.com/Carlosfengv.png
+- name: Chasen-Zhang
+  url: https://github.com/Chasen-Zhang
+  image_url: https://github.com/Chasen-Zhang.png
+- name: ClSlaid
+  url: https://github.com/ClSlaid
+  image_url: https://github.com/ClSlaid.png
+- name: drmingdrmer
+  url: https://github.com/drmingdrmer
+  image_url: https://github.com/drmingdrmer.png
+- name: dusx1981
+  url: https://github.com/dusx1981
+  image_url: https://github.com/dusx1981.png
+- name: everpcpc
+  url: https://github.com/everpcpc
+  image_url: https://github.com/everpcpc.png
+- name: johnhaxx7
+  url: https://github.com/johnhaxx7
+  image_url: https://github.com/johnhaxx7.png
+- name: mergify[bot]
+  url: https://github.com/apps/mergify
+  image_url: https://avatars.githubusercontent.com/in/10562
+- name: PsiACE
+  url: https://github.com/PsiACE
+  image_url: https://github.com/PsiACE.png
+- name: RinChanNOWWW
+  url: https://github.com/RinChanNOWWW
+  image_url: https://github.com/RinChanNOWWW.png
+- name: soyeric128
+  url: https://github.com/soyeric128
+  image_url: https://github.com/soyeric128.png
+- name: sandflee
+  url: https://github.com/sandflee
+  image_url: https://github.com/sandflee.png
+- name: sundy-li
+  url: https://github.com/sundy-li
+  image_url: https://github.com/sundy-li.png
+- name: TCeason
+  url: https://github.com/TCeason
+  image_url: https://github.com/TCeason.png
+- name: wubx
+  url: https://github.com/wubx
+  image_url: https://github.com/wubx.png
+- name: Xuanwo
+  url: https://github.com/Xuanwo
+  image_url: https://github.com/Xuanwo.png
+- name: xudong963
+  url: https://github.com/xudong963
+  image_url: https://github.com/xudong963.png
+- name: youngsofun
+  url: https://github.com/youngsofun
+  image_url: https://github.com/youngsofun.png
+- name: zhang2014
+  url: https://github.com/zhang2014
+  image_url: https://github.com/zhang2014.png
+- name: zhyass
+  url: https://github.com/zhyass
+  image_url: https://github.com/zhyass.png
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> :loudspeaker: Databend v1.0 is about to be released, and v1.0.0-nightly is already available as a candidate version. https://github.com/datafuselabs/databend/releases/tag/v1.0.0-nightly
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Support WebHDFS Backend
+
+HDFS is a widely used distributed file system for big data, and one of the storage backends supported by Databend. However, in previous implementations, using HDFS depended on Java environment and some specific jars, which might be inconvenient for some users.
```

Comment:
HDFS is a popular distributed file system for managing big data and is one of the storage backends supported by Databend. However, incorporating HDFS into Databend required a Java environment and specific jars, which were not suitable in some circumstances.